react-bootstrap/index.d.ts

```diff
@@ -590,6 +590,7 @@ declare namespace ReactBootstrap {
         active?: boolean;
         noCaret?: boolean;
         eventKey?: any;
+        title?: any; // TODO: Add more specific type
```

Comment:
`React.ReactNode`? I appreciate the `TODO`, but in practice that will never get fixed.